### ISSUE TITLE
Add cascade to drop statements in schema module

### ISSUE THREAD

**Issue by jaym910** [koskimas/kysely#143]
Hi I'm really liking the library. One thing I've found lacking: it seems that there is no ability to add a cascade modifier to a drop statement in the `schema` module.

I'm imagining it would work like the `ifExists` method and look something like:
 ```typescript
// modified typedef
DropTableBuilder implements OperationNodeSource, Compilable {
    #private;
    constructor(props: DropTableBuilderProps);
    cascade(): DropTableBuilder;
    ifExists(): DropTableBuilder;
    toOperationNode(): DropTableNode;
    compile(): CompiledQuery;
    execute(): Promise<void>;
}
// usage
dropTable('x').cascade()
```

Would this be something worth adding (I'd be willing to work on a PR). If you don't think it makes sense having this functionality, that's fine too--definitely not too hard to figure out a workaround within my own codebases.

I'd add it for each supported drop statement (table, schema, view, index)

**Comment by koskimas** [comment]
Hi! That's a good addition and I would definitely merge a PR.

**Comment by jaym910** [comment]
Cool, just made the PR: #144

**Comment by koskimas** [comment]
Thank you! Released in 0.21.4

**Comment by jaym910** [comment]
realized i totally forgot to commit my test cases so i opened another PR #145  incase you want them... no worries if you dont think they're neccessary
